tests/third_party/lua/src/Makefile

```diff
@@ -26,7 +28,7 @@ MYLIBS=
 MYOBJS=
 
 # XXX EMSCRIPTEN - add this
-AR_ARGS=rcu
+AR_ARGS=rc
```

Comment:
Why this?

Reply:
I don't remember the original reason for adding it to the makefile, but this change is because the native `ar` on my laptop errors if `u` is given, as an unrecognized flag. As this is just a line we added to the make file anyhow, I just did what the error asked for...